seeding/seeding.js

```diff
@@ -0,0 +1,96 @@
+console.log('Image Descripton Length: ', imageDesc.length);
+
+// Stay Name
+const stayName = ['Charming Craftsman East Sac/Midtown Apartment', 'Cozy Napa Wine Country Studio', 'The Cob house', 'Marina District Private Rooftop Studio with Deck', 'Serenity Seaside Garden Cottage (no cleaning fee)', 'Detached Wine Country "tiny house"', 'Sunset Yurt – stunning infinity pool', 'Downtown Cottage - Walk to Plaza!', 'Redwoods Treehouse', 'Bird Glamping', 'El Avistador. Montes de Toledo', 'Shepherd,s Hut with hot tub use near Stonehenge'];
+console.log('Stay Name Length: ', stayName.length);
+
+// Super Host
+const stayLocation = ['Sacramento', 'Angwin', 'Gualala', 'San Francisco', 'Santa Rosa', 'Garden Valley', 'Healdsburg', 'Sea Ranch', 'Sesquilé', 'Sonseca', 'DeLand Southwest', 'McKean', 'Fisher Island', 'Friendswood', 'Pewee Valley', 'Falling Waters', 'Anchor Bay', 'Poplar', 'Corona de Tucson', 'Longtown', 'Fidelity', 'Tropic', 'Pueblito del Carmen', 'Halstead', 'Tully', 'Miami Lakes', 'Maynard', 'Ocean City', 'Dillwyn', 'Lorenz Park', 'Fairacres', 'Hato Candal', 'King William', 'Springs', 'Bull Shoals', 'Upper Bear Creek', 'Watervliet', 'Grey Eagle', 'Kingston', 'Royal Lakes', 'Makakilo', 'Coffeen', 'Spangle', 'Alpine', 'Mango', 'Watsontown', 'Salmon', 'Ravalli', 'Lakewood Shores', 'Barahona', 'Locustdale', 'Dunnell', 'Chester Gap', 'North Fair Oaks', 'La Salle', 'Park Layne', 'Beechmont', 'Cornelia', 'Graceville', 'Trout Valley'];
+console.log('Stay Location Length: ', stayLocation.length);
+
+// Seed the DB with dummy data for react components to render out.
+
+// Random Int generator
+const randomInt = (min, max, floor) => {
+  if (floor) {
+    return Math.floor(min + Math.random() * (max - min));
+  }
+  return min + (Math.random() * (max - min));
+};
+
+// Super Host Boolean
+const superHostBool = () => {
+  const decision = randomInt(1, 100, true);
+  if (decision < 50) {
+    return true;
+  }
+  return false;
+};
+// console.log('Super Host Bool: ', superHostBool());
+
+// Rating Generator - This has to be a float
+const rating = () => randomInt(1, 5, true);
+// console.log('Random Rating: ', rating());
+
+// Review Count
+const reviewCount = () => randomInt(1, 300, true);
+// console.log('Review Count: ', reviewCount());
+
+// This will output an Array photo OBJs 10 of them for now.
+const imageSeeder = () => {
+  const url = 'www.aws.com/imageurl.jpg';
+  let result = [];
+  for (let count = 1; count < 10; count++) {
+    let obj = {
+      id: count,
+      photo_url: url,
+      description: imageDesc[randomInt(0, imageDesc.length, true)],
+    };
+    result.push(obj);
+  }
+  return result;
+};
+
+// Seed the DB
+const dataToSeed = () => {
+  let results = [];
+  let count = 1;
+
+  for (let i = 0; i < 100; i++) {
```

Comment:
Can refactor to use 2 lines (let count = 0) and use count instead of i. Or set roomId to i+1

Reply:
Fixed